CHANGELOG.md

```diff
@@ -7,6 +7,65 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 <!-- towncrier release notes start -->
 
+## [v4.2.0]
+
+### Released 2023-11-27
+
+### Breaking Changes
+
+- chore!: upgrade otel to 0.89.0-sumo-2 [#3368] [#3414]
+
+  Otelcol enqueue failure metrics are now only reported if they're non-zero. If you have monitors set based on the assumption that they're
+  always present, you'll need to update them. Monitors installed by this Chart are unaffected.
+
+### Added
+
+- feat(metrics): add setting affinity for metrics collector [#3400]
+
+### Changed
+
+- chore: add support for:
+  - EKS 1.28 [#3387]
+  - EKS Fargate 1.28 [#3387]
+  - OpenShift 4.14 [#3387]
+  - kops 1.28 [#3387]
+  - AKS 1.28 [#3387]
+  - GKE v1.28 [#3403]
+- chore: drop support for:
+  - EKS 1.23 [#3387]
+  - kops 1.23 [#3387]
+  - OpenShift 4.10 [#3387]
+- deps: update tailing-sidecar to 0.9.0 [#3391]
```

Comment:
I like this, was it generated automatically?

Reply:
Nope, I manually put in the sublists.